### ISSUE TITLE
feat(ui): add product tour

### ISSUE THREAD

**Issue by dev-bre** [switchfeat-com/switchfeat#15]
To help users to learn how to use SwitchFeat, a product tour should be implemented using Driver.js library.

**Comment by Ravina-Deogadkar** [comment]
Can I work on this?

**Comment by dev-bre** [comment]
yes of course! considering that the product is still at early stage, the UI is probably subject to change over time. is that ok for you?
